crates/chia-consensus/src/npc_result.rs

```diff
@@ -0,0 +1,72 @@
+use crate::allocator::make_allocator;
+use crate::consensus_constants::ConsensusConstants;
+use crate::gen::conditions::{
+    process_single_spend, validate_conditions, MempoolVisitor, ParseState, SpendBundleConditions,
+};
+use crate::gen::flags::MEMPOOL_MODE;
+use crate::gen::owned_conditions::OwnedSpendBundleConditions;
+use crate::gen::run_block_generator::subtract_cost;
+use crate::gen::validation_error::ValidationErr;
+use crate::multiprocess_validation::get_flags_for_height_and_constants;
+use chia_protocol::SpendBundle;
+use clvm_utils::{tree_hash_cached, TreeHash};
+use clvmr::allocator::{Allocator, NodePtr};
+use clvmr::chia_dialect::ChiaDialect;
+use clvmr::chia_dialect::LIMIT_HEAP;
+use clvmr::reduction::Reduction;
+use clvmr::run_program::run_program;
+use clvmr::serde::node_from_bytes;
+use std::collections::{HashMap, HashSet};
+
+pub fn get_name_puzzle_conditions(
+    spend_bundle: &SpendBundle,
+    max_cost: u64,
+    mempool_mode: bool,
```

Comment:
this parameter should be removed. You're only dealing with transactions, i.e. mempool mode

Reply:
Done